src/soilbiogeochem/SoilBiogeochemDecompCascadeMIMICSMod.F90

```diff
@@ -925,92 +926,77 @@ subroutine decomp_rates_mimics(bounds, num_soilc, filter_soilc, &
+            vmax_l2_m2 = exp(vslope_l2_m2 * t_soi_degC + vint_l2_m2) * vmod_l2_m2
+            vmax_s1_m1 = exp(vslope_s1_m1 * t_soi_degC + vint_s1_m1) * vmod_s1_m1
+            vmax_s1_m2 = exp(vslope_s1_m2 * t_soi_degC + vint_s1_m2) * vmod_s1_m2
 
             ! km are time-dependent terms
-            km_l1_m1 = exp(kslope_l1_m1 * t_soi24_degC_col(c,j) + kint_l1_m1) * kmod_l1_m1
-            km_l1_m2 = exp(kslope_l1_m2 * t_soi24_degC_col(c,j) + kint_l1_m2) * kmod_l1_m2
-            km_l2_m1 = exp(kslope_l2_m1 * t_soi24_degC_col(c,j) + kint_l2_m1) * kmod_l2_m1
-            km_l2_m2 = exp(kslope_l2_m2 * t_soi24_degC_col(c,j) + kint_l2_m2) * kmod_l2_m2
-            km_s3_m1 = exp(kslope_s3_m1 * t_soi24_degC_col(c,j) + kint_s3_m1) * kmod_s3_m1 / p_scalar(c,j)
-            km_s3_m2 = exp(kslope_s3_m2 * t_soi24_degC_col(c,j) + kint_s3_m2) * kmod_s3_m2 / p_scalar(c,j)
+            km_l1_m1 = exp(kslope_l1_m1 * t_soi_degC + kint_l1_m1) * kmod_l1_m1
+            km_l1_m2 = exp(kslope_l1_m2 * t_soi_degC + kint_l1_m2) * kmod_l1_m2
+            km_l2_m1 = exp(kslope_l2_m1 * t_soi_degC + kint_l2_m1) * kmod_l2_m1
+            km_l2_m2 = exp(kslope_l2_m2 * t_soi_degC + kint_l2_m2) * kmod_l2_m2
+            km_s1_m1 = exp(kslope_s1_m1 * t_soi_degC + kint_s1_m1) * kmod_s1_m1 / p_scalar(c,j)
+            km_s1_m2 = exp(kslope_s1_m2 * t_soi_degC + kint_s1_m2) * kmod_s1_m2 / p_scalar(c,j)
 
             ! Product of w_scalar * depth_scalar * o_scalar
             w_d_o_scalars = w_scalar(c,j) * depth_scalar(c,j) * o_scalar(c,j)
 
             ! decomp_k used in SoilBiogeochemPotentialMod.F90
-            ! TODO Defined correctly here?
-            ! TODO Combine multiple terms of same donor pools...
-            ! TODO Where to calculate m1_pool and m2_pool?
-            ! TODO When to introduce litter/soil biomass to the calculation?
-            ! TODO Excluded spinup terms for now. See cwd below how used.
-            decomp_k(c,j,i_met_lit) = (vmax_l1_m1 * m1_pool(c,j) / &
-                                       (km_l1_m1 + m1_pool(c,j))) * &
-                                      w_d_o_scalars
-            decomp_k(c,j,i_met_lit) = (vmax_l1_m2 * m2_pool(c,j) / &
-                                       (km_l1_m2 + m2_pool(c,j))) * &
+            ! TODO Excluded spinup terms for now (except see cwd below)
+            ! TODO Calc m*_conc = m*_pool(c,j) / dz(j) layer thickness
+            !           mgC/cm3 = gC/m3 * (1e3 mg/g) / (1e6 cm3/m3)
+            ! TODO decomp_k will end up in units of per hour and we
+            !      need per s so apply conversion to the coeffs above
+            decomp_k(c,j,i_met_lit) = (vmax_l1_m1 * m1_conc(c,j) / &
+                                       (km_l1_m1 + m1_conc(c,j)) + &
+                                       vmax_l1_m2 * m2_conc(c,j) / &
+                                       (km_l1_m2 + m2_conc(c,j))) * &
                                       w_d_o_scalars
```

Comment:
I'm not sure I follow...
Currently we set pathfrac = 1 for all transitions EXCEPT for
`m1s1, m1s2, m1s3`
`m2s1, m2s2, m2s3`
where we use
`fphys, fchem, faval`

As I think about it, we may need to change pathfrac for the others to avoid double-counting. So in your example
pathfrac(l1m1) = (vmax_l1_m1 * m1_conc(c,j) / (km_l1_m1 + m1_conc(c,j))) / (vmax_l1_m2 * m2_conc(c,j) / (km_l1_m2 + m2_conc(c,j)))
pathfrac(l1m2) = 1 - pathfrac(l1m1)
because doesn't this determine how much of the donor will go to each receiver? If we need to do this, we'll see how complicated it ends up being...

Reply:
I came up with a way to address this. I will commit and push changes by the end of the day.